src/components/widgets/NumberWidget.js

```diff
@@ -143,7 +139,6 @@ export default class NumberWidget extends React.Component {
 
     // enter.
     if (event.keyCode === 13) {
-      this.setValue(parseFloat(this.input.current.value));
```

Comment:
This is already done by the onBlur callback that's called with the blur() call below.